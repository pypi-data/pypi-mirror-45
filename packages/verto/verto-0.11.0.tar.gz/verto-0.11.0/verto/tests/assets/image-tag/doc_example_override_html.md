{image alt="dogs" file-path="http://placehold.it/350x150" alt="placeholder image" source="https://placehold.it/"}
