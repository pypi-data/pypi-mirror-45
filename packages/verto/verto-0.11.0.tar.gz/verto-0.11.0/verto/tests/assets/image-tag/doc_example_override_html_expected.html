<div class="text-center">
<img alt="placeholder image" class="rounded img-thumbnail" src="http://placehold.it/350x150" />
</div>
